fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -63575,6 +63574,260 @@ intersection_of: FBbt:00005106 ! neuron
+def: "A centrally located synaptic neuropil subdomain in larval abdominal segment 9 (A9)." [FBC:dw]
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00001756 ! larval abdominal segment 9
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T09:20:07Z
+
+[Term]
+id: FBbt:00007717
+name: central neuropil of larval tritocerebrum
+def: "A centrally located synaptic neuropil subdomain in larval tritocerebrum. It has a pyramidal shape with boundaries following the lateral curve of the CITd/v fascicles. It does not meet the central brain." [FBC:dw]
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00001962 ! embryonic/larval tritocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T09:20:07Z
+
+[Term]
+id: FBbt:00007718
+name: centromedial neuropil of larval tritocerebrum
+def: "A centro-medially located synaptic neuropil subdomain in larval tritocerebrum. The centro-medial tritocerebrum extends alongside the medial part of the anterior central brain." [FBC:dw]
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00001962 ! embryonic/larval tritocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T09:20:07Z
+
+[Term]
+id: FBbt:00007719
+name: dorsomedial neuropil of larval tritocerebrum
+def: "A dorso-medially located synaptic neuropil subdomain in larval tritocerebrum. It is the dorsal most of the 3 medial neuropils of the tritocerebrum which together make up all the neuropil between the medulla and the central brain." [FBC:dw]
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00001962 ! embryonic/larval tritocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T10:53:13Z
+
+[Term]
+id: FBbt:00007720
+name: ventromedial neuropil of larval tritocerebrum
+def: "A ventro-medially located synaptic neuropil subdomain in larval tritocerebrum. It is the dorsal most of the 3 medial neuropils of the tritocerebrum which together make up all the neuropil between the medulla and the central brain." [FBC:dw]
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00001962 ! embryonic/larval tritocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T10:53:13Z
+
+[Term]
+id: FBbt:00007721
+name: embryonic/larval superior inferior protocerebrum
+def: "A small synaptic neuropil of the larval brain that has the shape of a hemi-cylinder.  It flanks the mushroom body vertical lobe laterally and posteriorly,  where it wedges in between SLP (superior lateral protocerebrum) and SMPp. It borders the IPl and (briefly) IPa compartments and (very anteriorly and ventrally) the Spur." [FBC:dw]
+is_a: FBbt:00040007 ! synaptic neuropil domain
+relationship: part_of FBbt:00001925 ! embryonic/larval protocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T11:10:38Z
+
```

Comment:
removed typo and changed IPa>Crepine, IP>Clamp:
"A small primordium that appears as a ventrally directed process of the  lateral  primordial fan-shaped body.  It is demarcated from the surrounding medial clamp and crepine by different the different concentration of synapses (e.g. as assayed by Bruchpilot expression).  It develops from elements of the DPMm1, DPMpm1/2 and CM4 lineages."